src/components/App/App.css

```diff
@@ -5,6 +5,7 @@
 }
 
 html {
+  height: 100%;
   background-color: #191725;
   width: 100%;
```

Comment:
To update the html background colour save the property to a variable and add the colours in the :root
Then just toggle it with a useEffect that re-renders when the theme state changes

:root {
  --background-dark: #191725;
  --background-light: white;
}

html {
  height: 100%;
  background-color: var(--background);
  width: 100%;
}